site/docs/timeunit.md

```diff
@@ -51,34 +51,34 @@ A `timeUnit` transform in the `transform` array has the following properties:
 
 The `timeUnit` property of a channel definition sets the level of specificity for a temporal field. Currently supported values are:
 
-- Non-periodic Time Unit: `'year'`, `'yearmonth'`, `'yearmonthday'`, `'yearmonthdate'`, `'yearday'`, `'yeardate'`, `'yearmonthdayhours'`, `'yearmonthdayhoursminutes'`, `'utcyear'`, `'utcyearmonth'`, `'utcyearmonthday'`, `'utcyearmonthdate'`, `'utcyearday'`, `'utcyeardate'`, `'utcyearmonthdayhours'`, `'utcyearmonthdayhoursminutes'`.
+- Non-periodic Time Unit: `'year'`, `'yearmonth'`, `'yearmonthday'`, `'yearmonthdate'`, `'yearday'`, `'yeardate'`, `'yearmonthdayhours'`, `'yearmonthdayhoursminutes'`
   - For example the `yearmonth` of `April 4, 2016 11:52:34` is `April 2016`.
 
-- Periodic Time Unit: `'month'`, `'day'`, `'date'`, `'hours'`, `'minutes'`, `'seconds'`, `'milliseconds'`,  `'hoursminutes'`, `'hoursminutesseconds'`, `'minutesseconds'`, `'secondsmilliseconds'`, `'utcmonth'`, `'utcday'`, `'utcdate'`, `'utchours'`, `'utcminutes'`, `'utcseconds'`, `'utcmilliseconds'`,  `'utchoursminutes'`, `'utchoursminutesseconds'`, `'utcminutesseconds'`, `'utcsecondsmilliseconds'`.
+- Periodic Time Unit: `'month'`, `'day'`, `'date'`, `'hours'`, `'minutes'`, `'seconds'`, `'milliseconds'`,  `'hoursminutes'`, `'hoursminutesseconds'`, `'minutesseconds'`, `'secondsmilliseconds'`,
```

Comment:
My vote would be to get rid of it. I think the `timeUnit` values by themselves are sufficiently descriptive that the periodic/non-periodic classification hampers understandability.